src/scss/components/menu/_index.scss

```diff
@@ -44,7 +44,7 @@ hx-menuitem {
   background-color: transparent;
   border-radius: 2px;
   border: none;
-  color: $cyan-900;
+  color: var(--hxMenu-hxMenuitem-color, $cyan-900);
```

Comment:
Please update color variable to Color Palette 2.0 specs.

Reply:
Changes have been done, Please review